test/parallel/test-dgram-send-bad-arguments.js

```diff
@@ -20,17 +20,25 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const dgram = require('dgram');
 
 const buf = Buffer.from('test');
 const host = '127.0.0.1';
 const sock = dgram.createSocket('udp4');
 
-assert.throws(() => {
-  sock.send();
-}, TypeError);  // First argument should be a buffer.
+// First argument should be a buffer.
+assert.throws(() => { sock.send(); }, TypeError);
```

Comment:
Nit: we don't need both this and `common. expectsError()`, I'd keep only `common. expectsError()`. Same below.

Reply:
I'll update accordingly. Thank you.